Doc/whatsnew/3.14.rst

```diff
@@ -1518,6 +1518,13 @@ io
   :gh:`120754` and :gh:`90102`.)
 
 
+json
+----
+
+* Improve the performance of :class:`~json.JSONEncoder` encodes strings.
+  (Contributed by Inada Naoki in :gh:`133186`.)
```

Comment:
You should now retarget this change to Python 3.15 (move the text to Doc/whatsnew/3.15.rst).